pkg/pointerdb/pdbclient/client.go

```diff
@@ -65,40 +66,34 @@ type Client interface {
 
 // NewClient initializes a new pointerdb client
 func NewClient(identity *provider.FullIdentity, address string, APIKey string) (*PointerDB, error) {
-	dialOpt, err := identity.DialOption()
-	if err != nil {
-		return nil, err
-	}
-
 	signatureHeader := &metadata.MD{}
-	peer := &peer.Peer{}
-	apiKeyInjector := grpcauth.NewAPIKeyInjector(APIKey, grpc.Header(signatureHeader), grpc.Peer(peer))
-	c, err := clientConnection(address, dialOpt, grpc.WithUnaryInterceptor(apiKeyInjector))
-
+	_peer := &peer.Peer{}
```

Comment:
`peer` here was fine.

Reply:
`peer` is another package name from grpc

Reply:
this is another package name collision

Reply:
I'm not following. I guess the main nit is about avoiding underscores in front of variables, it doesn't look nice. I think having a variable `peer` here didn't cause problems, unless I missed something. Of course, there's possibility to use different name altogether, e.g. `peerInfo` or `peerAddress` etc.

Reply:
This part (`peer` variable) is removed in current master, no need for discussion :)

Reply:
the var `peer` would collide with [grpc's `peer`](https://godoc.org/google.golang.org/grpc/peer) package: `peer := &peer.Peer{}`

Reply:
It didn't end up mattering because `peer` was removed from the struct and the logic.